bottlenecks.md

```diff
@@ -11,189 +11,255 @@ kernelspec:
+plt.plot(bottBlist, prob2, c='blue', marker='+', lw=1, ms=8, label=f"T={bottT2}, simulated")
+plt.xlabel("Bottleneck Strength B")
+plt.ylabel("p(t>T)")
+plt.legend()
+plt.show()
+
+plt.plot(bottBlist, expMean, c='brown', ls=":", lw=4, label=f"T={bottT}, theoretical")
+plt.plot(bottBlist, dat, c='red', marker='+', ms=8, label=f"T={bottT}, simulated")
+plt.plot(bottBlist, expMean2, c='navy', ls=":", lw=4, label=f"T={bottT2}, theoretical")
+plt.plot(bottBlist, dat2, c='blue', marker='+', ms=8, label=f"T={bottT2}, simulated")
+plt.xlabel("Bottleneck Strength B");
+plt.ylabel("E[t]")
+plt.legend()
+plt.show()
 ```
 
+## The distribution of pairwise coalescence times
+
+The distribution of pairwise coalescence times has two maxima at $t=0$ and the bottleneck time $t=T$ (we have assumed $T=0.8$ below) as expected. The simulated distribution of pairwise coalescence times fits the analytic expectation:
+
 ```{code-cell} ipython3
-fig, ax = plt.subplots()
-index = np.arange(1,4)
-bar_width = 0.4
-print(data4[0][1:])
-print(data4[1][1:])
-print(data4[2][1:])
-print(data4[3][1:])
+s = bottBlist[3]
+sprob = 1-np.exp(-s)
+
+coaldis = pairCoalBott(s, bottT2, 10000);
+coaldisFilt = coaldis[(coaldis < 3)];
+
+probtest = len(coaldis[coaldis>bottT2])/10000
+expprob2 = np.exp(-(bottT2+s))
 
-simsfs = ax.bar(index, data4[3][1:], bar_width, alpha=opacity, label='sim')
-expextsfs = ax.bar(index+ bar_width, expsfsBottlN, bar_width, label='exp')
+tlist = np.arange(0.0,3,0.25);
+coalEpx1 = [np.exp(-i) for i in tlist[0:4]];
+coalEpx2 = [np.exp(-i)-(sprob*np.exp(-i)) for i in tlist[3:]];
 
-fig.tight_layout()
+plt.plot(tlist[0:4], coalEpx1, color = 'black', linewidth = 2);
+plt.plot(tlist[3:], coalEpx2, color = 'black', linewidth = 2);
+plt.hist(coaldisFilt, bins = 20, density=True);
+plt.xlabel("t");
+plt.ylabel("f(t)")
 plt.show()
 ```
 
-## The distribution of nton branches
+## Approximating the site frequency spectrum
+
+Bottlenecks can have a substantial effect on the site frequency spectrum (SFS). The SFS is a fundamental summary of sequence variation that forms the basis of many modern inference approaches (e.g. sweepfinder, DFE-alpha, dadi). in the absence of linkage information the SFS is a lossless summary, i.e. any summary of sequence variation that ignores linkage (e.g. pairwise measures of diversity and divergence, $F_{st}$, Tajima's D etc) are summaries of the SFS (Achaz 2009). The SFS is convenient analytically, since it depends only on the mean length and frequency of genealogical branches. For many demographic models of interest the mean length of nton branches can be derived either using coalescent theory (Chen 2011) or diffusion equations (Gutenkunst 2009). A number of composite likelihood approaches have been developed based on either analytic results for the SFS or approximate obtained from coalescent simulations (Gutenkunst 2009, Excoffier 2002). We can use msprime to obtain the SFS for a sample of $n=20$ for a range of bottleneck strengths:
```

Comment:
```suggestion
Bottlenecks can have a substantial effect on the site frequency spectrum (SFS). The SFS is a fundamental summary of sequence variation that forms the basis of many modern inference approaches (e.g. sweepfinder, DFE-alpha, dadi). in the absence of linkage information the SFS is a lossless summary, i.e. any summary of sequence variation that ignores linkage (e.g. pairwise measures of diversity and divergence, $F_{st}$, Tajima's D etc) are summaries of the SFS (Achaz 2009). The SFS is convenient analytically, since it depends only on the mean length and frequency of genealogical branches. For many demographic models of interest the mean length of n-ton branches can be derived either using coalescent theory (Chen 2011) or diffusion equations (Gutenkunst 2009). A number of composite likelihood approaches have been developed based on either analytic results for the SFS or approximate obtained from coalescent simulations (Gutenkunst 2009, Excoffier 2002). We can use msprime to obtain the SFS for a sample of $n=20$ for a range of bottleneck strengths:
```

Reply:
We should probably replace "site frequency spectrum" by "allele frequency spectrum" anyway, since that's what we use in the tskit docs